src/pam_sddm/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+include(KDEInstallDirs)
+
+find_package(PAM REQUIRED)
+find_package(keyutils REQUIRED)
```

Comment:
```
find_package(keyutils REQUIRED)
add_definitions(-DHAVE_KEYUTILS)
```

something here is off. We're making this required, yet have a code path to make it optional